### ISSUE TITLE
Syntax highlighting support for .mly/.mll files

### ISSUE THREAD

**Issue by Chillee** [reasonml-editor/vscode-reasonml#163]
It's a pretty common file format for people working with ocaml writing compilers,  and the deprecated ocaml plugin has support for it.

**Comment by zachbai** [comment]
+1 would like this feature

**Comment by fstiffo** [comment]
+1 the old plugin vscode-ocaml had the support to *,mll and *.mly. Please reintroduce it.

**Comment by maelvalais** [comment]
Maybe we could copy the tmLanguage files from the old extension to the new one in the meantime?

**Comment by juanbono** [comment]
I [did](https://github.com/juanbono/vscode-reasonml/tree/ocamllex-ocamlyacc-support) what @maelvalais said but I only got ocamlyacc/Menhir working. I'll take a look again later :)

**Comment by freebroccolo** [comment]
I'm planning to release an update with support for this soon.

**Comment by juanbono** [comment]
Great! 
PS: Thank you @freebroccolo for all the work you have done!

**Comment by agustingianni** [comment]
@freebroccolo any updates on this?

**Comment by giltho** [comment]
@freebroccolo hi, if you need any help on this let me know :)

**Comment by chengtie** [comment]
any update or any workaround?

**Comment by giltho** [comment]
I don't think this repo is maintained anymore, I have a pending pull request that's a year old
